bin/compile.js

```diff
@@ -15,6 +15,7 @@ program
     .option('-w, --watch', 'watch for changes and recompile')
     .option('-b, --bytecode', 'output bytecode')
     .option('-x, --no-babelify', 'skip Babel transforms')
+    .option('-S, --no-sourcemap', 'disable sourcemap in browserify')
```

Comment:
How about `'omit sourcemap'`? I think it's better to avoid mentioning internal implementation details.

Reply:
updated the comment